src/main/resources/application.properties

```diff
@@ -1 +1,6 @@
-
+spring.datasource.url=jdbc:mysql://localhost:3306/book_store?serverTimezone=UTC
```

Comment:
hint: you might find `.yml` format better than `.properties` 

but it is just a suggestion - no need to fix anything :)